tests/test_mgxs_library_distribcell/results_true.dat

```diff
@@ -1,8 +1,11 @@
                                     avg(distribcell)  group in nuclide      mean  std. dev.
-0  (0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13,...         1   total  0.718919   0.520644                                    avg(distribcell)  group in nuclide  mean  std. dev.
-0  (0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13,...         1   total   0.0        0.0                                    avg(distribcell)  group in  group out nuclide moment      mean  std. dev.
+0  (0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13,...         1   total  0.718919   0.520644
```

Comment:
I simply added an endl after each df print in these test_mgxs_library* tests because when those results_true.dat were visually not useful.  The header for the next dataset was on the same line as the previous data set.